week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,16 +7,30 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
-
+function journeyPlanner(londonLocations,transportType) {
```

Comment:
Like with the last two, this looks like it's really over-fitting to the test cases, rather than the intent of the problem - can you try again? We can help you out on slack if you get stuck :)